Sources/Components/BroadcastView/BroadcastView.swift

```diff
@@ -4,23 +4,32 @@
 
 import UIKit
 
+public protocol BroadcastViewDelegate: class {
+    func broadcastViewDismissButtonTapped(_ broadcastView: BroadcastView)
+    func broadcastView(_ broadcastView: BroadcastView, didTapURL url: URL)
+}
+
 /// Broadcast messages appears without any action from the user.
 /// They are used when it´s important to inform the user about something that has affected the whole system and many users.
 /// Especially if it has a consequence for how he or she uses the service.
 /// https://schibsted.frontify.com/d/oCLrx0cypXJM/design-system#/components/broadcast
-public class BroadcastView: UIView {
-    private lazy var messageLabel: Label = {
-        let label = Label(style: Style.labelStyle)
-
-        label.translatesAutoresizingMaskIntoConstraints = false
-        label.numberOfLines = 0
-
-        return label
+public final class BroadcastView: UIView {
+    private lazy var messageTextView: UITextView = {
+        let textView = UITextView()
+        textView.delegate = self
+        textView.translatesAutoresizingMaskIntoConstraints = false
+        textView.backgroundColor = .clear
+        textView.isAccessibilityElement = true
+        textView.isEditable = false
+        textView.isSelectable = true
+        textView.isScrollEnabled = false
+        textView.textContainerInset = .zero
+        textView.linkTextAttributes = BroadcastView.Style.linkTextAttributes
```

Comment:
I don't know if it's better or if it would work at all, but could you have set the `textView.dataDetectorTypes = .link` to get the same results or is this a custom type of link style?

Reply:
The `textView.dataDetectorTypes = .link` will make urls in the text in the `UITextView` tappable, yes 😊